### PR TITLE
Typo in Docker - introduction

### PR DESCRIPTION
**Pull request by seangwright**
## Summary

Describe your changes here.

Removed extra `to create`

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/docker/introduction.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The Microsoft Container Registry (MCR) is the official source of Microsoft-provi
 
 ### Dockerfile
 
-A **Dockerfile** is a file that defines a set of instructions that creates an image. Each instruction in the **Dockerfile** creates a layer in the image. For the most part, when you rebuild the image only the layers that have changed are rebuilt. The **Dockerfile** can be distributed to others and allows them to recreate to create a new image in the same manner you created it. While this allows you to distribute the *instructions* on how to create the image, the main way to distribute your image is to publish it to a registry.
+A **Dockerfile** is a file that defines a set of instructions that creates an image. Each instruction in the **Dockerfile** creates a layer in the image. For the most part, when you rebuild the image only the layers that have changed are rebuilt. The **Dockerfile** can be distributed to others and allows them to recreate a new image in the same manner you created it. While this allows you to distribute the *instructions* on how to create the image, the main way to distribute your image is to publish it to a registry.
```
Comment (Youssef1313) [inline docs/core/docker/introduction.md]:
```suggestion
A **Dockerfile** is a file that defines a set of instructions that creates an image. Each instruction in the **Dockerfile** creates a layer in the image. For the most part, when you rebuild the image, only the layers that have changed are rebuilt. The **Dockerfile** can be distributed to others and allows them to recreate a new image in the same manner you created it. While this allows you to distribute the *instructions* on how to create the image, the main way to distribute your image is to publish it to a registry.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you for making this improvement. Would you mind applying @Youssef1313's suggestion as well?

File: docs/core/docker/introduction.md (hunk 1 above)
Comment (seangwright) [reply docs/core/docker/introduction.md]: Added!

**Comment by seangwright** [comment]
@mairaw 👍, added it!

**Review by mairaw — APPROVED** [review]
Thanks!

**Pull request merged**